scripts/buidl.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+# Script for building a host with injected json data
+
+set -o pipefail
+trap cleanup EXIT
+trap cleanup SIGINT
+
+data_nix="/tmp/data.nix"
```

Comment:
We should use some auto-generated tmp folder and or a filename. See `mktmp` command. I also wonder can we substitute `/tmp` with something akin to XDG directory (https://wiki.archlinux.org/title/XDG_Base_Directory). I guess `/tmp` always exists but it could live in some other folder -- by the least, we want to give a possibility to set an environment variable to override the directory in which this file exists

Reply:
There is an issue with auto-generating temporary stuff using `mktemp` command in this use case. The problem is that the `flake.nix` file requires a hard-coded path to import this file into the host if it exists. Therefore, the path cannot be randomized in any way. This also prevents the user from casually overriding the path just through environment variables.

Reply:
In that case then it's fine, let's circle back to this if it ever becomes an issue.